File: near-pharma-license-nft/ZK-Medical-Wallet/Panacea-Wallet/UniWallet/app/js/unilib.js
var walletURL = "http://127.0.0.1:8080/";
function doProcess(theUrl,callback){
    var xhttp = new XMLHttpRequest();
    xhttp.onreadystatechange = function() {
        if (this.readyState == 4 && this.status == 200) {
           //console.log(xhttp.responseText);
           callback(null,xhttp.responseText);
        }
    };
    xhttp.open("GET", theUrl, true);
    xhttp.send();
}
function sendTransaction(accounToTransfer, amount, callback){
    var sendURL = `${walletURL}tx/sendTransaction?accountToTransfer=${accounToTransfer}&amount=${amount}`;
    doProcess(sendURL,(err,res)=>{
        if(!err){
            callback(null,res);
        }
    });
}
function contractFunctionTransaction(abi,contractAddress,contractFunction,callback){
    var sendURL = `${walletURL}tx/contractFunction?abi=${abi}&contractAddress=${contractAddress}&contractFunction=${contractFunction}`;
    doProcess(sendURL,(err,res)=>{
        if(!err){
            callback(null,res);
        }
    });
}
function contractFunctionTransactionCall(abi,contractAddress,contractFunction,callback){
    var sendURL = `${walletURL}tx/contractFunction/call?abi=${abi}&contractAddress=${contractAddress}&contractFunction=${contractFunction}`;
    doProcess(sendURL,(err,res)=>{
        if(!err){
            callback(null,res);
        }
    });
}
function checkWallet(callback){
    var sendURL = `${walletURL}testWallet`;
    doProcess(sendURL,(err,res)=>{
        callback(null,res);
    });
}

File: pharma-license-nft-near/ZK-Medical-Wallet/Panacea-Wallet/python/unilib.py
import requests
walletUrl = "http://localhost:8080"
# r = requests.get(walletUrl+"/testWallet")
# print(r.content)

# r = requests.get(walletUrl+"/tx/sendTransaction?accountToTransfer=0xffcf8fdee72ac11b5c542428b35eef5769c409f0")
# print(r.content)
def testWallet():
    r = requests.get(walletUrl+"/testWallet")
    return r.content.decode('utf8')

def sendTransaction(accountToSend):
    r = requests.get(walletUrl + "/tx/sendTransaction?accountToTransfer="+accountToSend)
    return r.content.decode('utf8')
def contractFunction(abi,contractAddress,contractFunction):
    r = requests.get(walletUrl+"/tx/contractFunction?abi="+abi+"&contractAddress="+contractAddress+"&contractFunction="+contractFunction)
    return r.content.decode('utf8')
def contractFunctionCall(abi,contractAddress,contractFunction):
    r = requests.get(walletUrl+"/tx/contractFunction/call?abi="+abi+"&contractAddress="+contractAddress+"&contractFunction="+contractFunction)
    return r.content.decode('utf8')